templates/artist/includes/artist_detail.html

```diff
@@ -0,0 +1,6 @@
+{% load thumbnail %}
```

Comment:
Yeah, I just saw the other PR. Makes sense. You should be able to just delete the file `templates/artist/includes/artist_detail.html` from this branch and all should be good.